packages/messaging/README.md

```diff
@@ -201,6 +201,56 @@ for (let item in proposition.items) {
 }
 ```
 
+
+### Unified track
+
+A unified tracking API is available for any proposition item (e.g., Content Cards, HTML, JSON, code-based items). Prefer calling `track` directly on the item.
+
+#### Using PropositionItem.track (recommended)
+
+**Syntax**
+
+```javascript
+// Overload 1: event only
+propositionItem.track(MessagingEdgeEventType.DISPLAY);
+
+// Overload 2: interaction + event + optional tokens
+propositionItem.track(
+  interaction /* string | null */, 
+  MessagingEdgeEventType.INTERACT /* enum value */, 
+  [/* tokens */] /* string[] | null */
+);
+```
+
+When using `getPropositionsForSurfaces`, the returned objects can be wrapped with `MessagingProposition` to get typed items and convenient tracking via `PropositionItem.track(...)`.
+
+```javascript
+import { Messaging, MessagingProposition, MessagingEdgeEventType } from '@adobe/react-native-aepmessaging';
+
+const SURFACES = ['mobileapp://my-surface'];
+const messages = await Messaging.getPropositionsForSurfaces(SURFACES);
+
+for (const surface of SURFACES) {
+  const propositions = messages[surface] || [];
+
+  for (const proposition of propositions) {
+    const msgProp = new MessagingProposition(proposition);
+
+    if (msgProp.items.length > 0) {
+      const propositionItem = msgProp.items[0];
+
+      // Overload 1: event only
```

Comment:
Remove  // Overload 1: event only

Use
// Track display event

(We have mentioned Overload 1 in the syntax, we can skip it in the example)